git_machete/annotation.py

```diff
@@ -13,8 +13,10 @@ def __init__(self, annotation: str):
         self._annotation_without_qualifiers = annotation
         self._rebase_text = ''
         self._push_text = ''
+        self._slide_out_text = ''
         self.rebase = True
         self.push = True
+        self.slide_out = None
```

Comment:
`True`?

Reply:
@PawelLipski  didn't want to assume `True` since traverse by default doesn't slide-out the branch, it does so only when certain conditions are met.

Reply:
>  it does so only when certain conditions are met.

Hmmm but the same holds true of rebase and push. The case when slide-out happens might be less frequent than the case for rebase or push... but still, none of these 3 action happens _by default_

Reply:
So I can't see a reason to treat slide-out differently!

Reply:
@PawelLipski Yes, you're right, but still, i don't like the way it would work when assuming `True` value in this example:
1. The conditions were not met so `needs_slide_out` value is `False`, annotations are present for that branch (e.g. `rebase=no`) so it would assign `True` to `needs_slide_out` which would be wrong IMO. To avoid this i added extra check + default `None` value.

Reply:
> annotations are present for that branch (e.g. rebase=no) so it would assign True to needs_slide_out

Hmmm for clarity... how would the existence of `rebase=no` qualifier (as opposed to `slide-out=no`) cause `needs_slide_out` to be `True`? 🤔

Reply:
Well, using logic used in existing qualifiers i.e. 
```
if branch in self.annotations:
      needs_slide_out = self.annotations[branch].qualifiers.slide_out
```
would cause `needs_slide_out` to be set `True` if any of the three qualifiers were present (it would't even need to be `slide-out=no`) and that would be wrong, @PawelLipski do you see it now?